### PR TITLE
Add io.github.dzheremi2.lrcmake-gtk

### PR DESCRIPTION
**Pull request by Dzheremi2**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly. 
Syncing songs lyrics using timestamps and exporting it to LRCLib
- [x] The domain used for the application ID is [controlled by the application developer(s)][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by Dzheremi2** [comment]
bot, build io.github.dzheremi2.lrcmake_gtk

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: why this restruction?

File: io.github.dzheremi2.lrcmake_gtk.json [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "lrcmake",
```
Comment (hfiguiere) [inline io.github.dzheremi2.lrcmake_gtk.json:28]: this module should be last since dependencies come first.

File: flathub.json (hunk 1 above)
Comment (Dzheremi2) [reply flathub.json:2]: I'm not sure that app will run on aarch64

Comment (hfiguiere) [reply flathub.json:2]: it's clear that we'll never know.

**Comment by Dzheremi2** [comment]
bot, build io.github.dzheremi2.lrcmake_gtk

**Comment by Dzheremi2** [comment]
bot, build io.github.dzheremi2.lrcmake_gtk

**Comment by Dzheremi2** [comment]
@hfiguiere I've got this error `appid-url-not-reachable`
Seems that it's happening because I used `lrcmake_gtk` in `APP_ID`, but repo is located under `lrcmake-gtk`
Docs say that I shouldn't use hyphen and should use underscore instead.

**Comment by bbhtt** [comment]
It says nit to use hyphen in the _domain_ portion of the ID.

**Comment by hfiguiere** [comment]
> @hfiguiere I've got this error `appid-url-not-reachable` Seems that it's happening because I used `lrcmake_gtk` in `APP_ID`, but repo is located under `lrcmake-gtk` Docs say that I shouldn't use hyphen and should use underscore instead.

It's say that, for all but the **last** component.

**Comment by Dzheremi2** [comment]
bot, build io.github.dzheremi2.lrcmake_gtk

**Comment by Dzheremi2** [comment]
bot, build io.github.dzheremi2.lrcmake_gtk

File: io.github.dzheremi2.lrcmake-gtk.json [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+                            "type": "file",
+                            "url": "https://files.pythonhosted.org/packages/ce/d9/5f4c13cecde62396b0d3fe530a50ccea91e7dfc1ccf0e09c228841bb5ba8/urllib3-2.2.3-py3-none-any.whl",
+                            "sha256": "ca899ca043dcb1bafa3e262d73aa25c465bfb49e0bd9dd5d59f1d0acba2f8fac"
+                        }
+                    ]
+                }
+	        ]
+        },
+        {
+            "name": "blueprint-compiler",
```
Comment (hfiguiere) [inline io.github.dzheremi2.lrcmake-gtk.json:122]: this is missing the cleanup rule to remove it all from the build at the end.

**Pull request closed without merging**